server/routerlicious/packages/services/src/mongoExceptionRetryRules/defaultRule.ts

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Lumberjack } from "@fluidframework/server-services-telemetry";
+import { IMongoExceptionRetryRule } from "./IMongoExceptionRetryRule";
+
+export class DefaultRule implements IMongoExceptionRetryRule {
```

Comment:
nit: DefaultMongoExceptionRule? DefaultRule feels a bit too generic, unless this will not be exposed outside of this folder.